### DISCUSSION TITLE
How to implement continuous speech recognition same as other speech recognition and display it as streaming?

### DISCUSSION THREAD

**Discussion by Bruce337f** [openai/whisper#1832] [Q&A]
I checked most of the discussion topics about **continuous recognition** and found that they are basically about recording audio on the fly through a microphone to return streaming recognition results, but there is no continuous recognition method for the audio file itself.
I want to achieve the same continuous recognition effect as Microsoft speech recognition. What do I need to modify?

**Comment by phineas-pta** [comment]
u shouldn't

whisper work best with 30s audio input
